pomodoro-cli.sh

```diff
@@ -45,7 +65,7 @@ function minutes_to_seconds() {
 
 ### Add minutes to current time
 ## Params: minutes {Number} - Ammounts of minutes to be added
-## Returns: {Date} - Current date plus minutes
+#Envia novos funcionários# Returns: {Date} - Current date plus minutes
```

Comment:
what's this?

Reply:
Removing the unnecessary comment. Not sure how that got there